components/InputFormGuest.js

```diff
@@ -11,11 +11,57 @@ export default function AddGuest() {
           <Input
             className="SmallFontStyle"
             type="text"
-            id="contactData"
-            name="contactData"
+            id="firstName"
+            name="firstName"
             required
+            placeholder="Vorname"
+          />
+          <Input
+            className="SmallFontStyle"
+            type="text"
+            id="lastName"
+            name="lastName"
+            required
+            placeholder="Nachname"
+          />
+          <Input
+            className="SmallFontStyle"
+            type="text"
+            id="address"
+            name="address"
+            required
+            placeholder="Anschrift"
+          />
+          <label className="MediumFontStyle" htmlFor="companion">
+            Begleitpersonen
+          </label>
+          <StyledDiv className="SmallFontStyle">Weiterer Gast</StyledDiv>
+          <StyledDiv className="SmallFontStyle">Kinder</StyledDiv>
+          <label className="MediumFontStyle" htmlFor="group">
+            Gruppe
+          </label>
+          <Input
+            className="SmallFontStyle"
+            type="text"
+            id="group"
+            name="group"
+            placeholder="z.B. Freunde der Braut"
+            required
+          />
+          <label className="MediumFontStyle" htmlFor="notes">
+            Anmerkungen
+          </label>
+          <InputNotes
+            className="SmallFontStyle"
+            type="text"
+            id="notes"
+            name="notes"
+            placeholder="Habt ihr Allergien, Unverträglichkeiten etc.? Lasst es uns bitte wissen :)"
           />
         </Fieldset>
+        <Button type="submit" className="MediumFontStyle" value="Speichern">
```

Comment:
I am not sure, but do you really need the value attribute here?

Reply:
I did some input fields with my coach and she told me to use the value so used it everywhere. I´ll check on that again.